e2e/app/component/admin-table.ts

```diff
@@ -15,83 +15,44 @@ export default class adminTable extends Table {
     return this.userLinkElement.asElement();
   }
 
-  getFrozenHeader(): Table {
+  getHeaderRow(): Table {
     return new Table(
       this.page,
       `${this.getXpath()}
-      //div[@class="p-datatable-scrollable-view p-datatable-frozen-view"]
-      //table[@class="p-datatable-scrollable-header-table"]`
+      //div[@class="p-datatable-wrapper"]
+      //table[@class="p-datatable-table"]
+      //thead//tr`
     );
   }
 
-  getFrozenBody(): Table {
+  getTable(): Table {
     return new Table(
       this.page,
       `${this.getXpath()}
-      //div[@class="p-datatable-scrollable-view p-datatable-frozen-view"]
-      //table[@class="p-datatable-scrollable-body-table"]`
-    );
-  }
-
-  getHeaderTable(): Table {
```

Comment:
can also remove header and footer from from `data-table.ts` - it looks like getHeaderTable() is used there, but its caller `dataTableSelectAllRows()` is also not used, so can also be removed.